Branch-SDK/Branch-SDK/Branch.h

```diff
@@ -154,6 +155,33 @@ typedef NS_ENUM(NSUInteger, BranchCreditHistoryOrder) {
  */
 + (Branch *)getInstance:(NSString *)branchKey;
 
+/// TODO: Document:
```

Comment:
Did you consider using class properties here? I like using `@property (class, readonly) BOOL useTestKey` because of the way it's bridged to Swift. With a class property you have `Branch.useTestBranchKey` in Swift. With a class method you have `Branch.useTestBranchKey()`. This is a minor concern. I'm mainly curious if you have any reason not to use class properties, maybe because of Xcode 7 support or something.